staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -601,7 +601,7 @@ type KubeletConfiguration struct {
 	// the node is recommended before changing this field.
 	// Default: "/etc/resolv.conf"
```

Comment:
the behavior of providing an explicit empty string needs to be described in the field doc

Reply:
Sure!
Added description according to this https://github.com/kubernetes/kubernetes/blob/9c7426e14f977026ed74dce0c537ebb486f15cd7/pkg/kubelet/network/dns/dns.go#L401-L422